### PR TITLE
Hackathon debug branch

### PR DESCRIPTION
**Pull request by HuyueLiRH**
This is the wrap up for everyone's demo , To run this, you need to download the model_data.zip file and unzip it to /data folder

### CODE REVIEW

File: streamlit/pages/top_bottom_20.py
Comment (tylncck) [inline streamlit/pages/top_bottom_20.py:1]: top_bottom_20.py created for Top Bottom 20 subpage of Streamlit App: 
I got following error in the first plot: DriverError: notebooks//output.geojson: No such file or directory

Due to these errors, I couldn't check the rest of streamlit subpage. I think the required data files are not in the correct directory.

File: streamlit/pages/box_plot.py
Comment (tylncck) [inline streamlit/pages/box_plot.py:1]: box_plot.py file created for Box Plot subpage of Streamlit App: 
Got following error in the first plot 
DriverError: data/features_table.gpkg: No such file or directory

Due to these errors, I couldn't check the rest of streamlit subpage. I think the required data files are not in the correct directory.

Comment (HuyueLiRH) [reply streamlit/pages/box_plot.py:1]: New path just updates, could you check again, Sorry about the inconvenience

**Review by AbhishekDand — APPROVED** [review]
Works nice and fast.. Thanks

**Comment by SoroushMaleki** [comment]
Thanks Huyue and Amazing work guys! It all looks great!

**Pull request merged**